gpcontrib/arenadata_toolkit/sql/arenadata_toolkit_test.sql

```diff
@@ -0,0 +1,106 @@
+----------------------------------------------------------------------------------------------------------
+-- test helpers
+----------------------------------------------------------------------------------------------------------
+SET client_min_messages=WARNING;
+-- function that mocks manual installation of arenadata_toolkit from bundle
+CREATE FUNCTION mock_manual_installation() RETURNS VOID AS $$
+DECLARE
+	dt timestamp without time zone;
+	partname VARCHAR(7);
+	startdate VARCHAR(19);
+	enddate VARCHAR(19);
+BEGIN
+	dt = NOW();
+	partname = 'p' || to_char(dt, 'YYYYMM');
+	startdate = to_char(dt, 'YYYY-MM-01');
+	enddate = to_char(dt + interval '1 month','YYYY-MM-01');
+
+	CREATE SCHEMA arenadata_toolkit;
+	EXECUTE 'GRANT ALL ON SCHEMA arenadata_toolkit to ' || (SELECT current_user);
```

Comment:
Why do we need a subquery here and below?
```sql
select replace(current_user, 'po', '');
 replace 
---------
 stgres
(1 row)
```

Reply:
I meant to write simply:
```sql
	EXECUTE 'GRANT ALL ON SCHEMA arenadata_toolkit to ' || current_user;
```

Reply:
fixed